pages/calendarPage/calendarPageBarber.js

```diff
@@ -0,0 +1,316 @@
+    { label: 'Yaron 10:00', value: '2' },
+    { label: 'Sapir 10:30', value: '3' },
+    { label: 'Bar 11:00', value: '4' },
+    { label: 'Tal 11:30', value: '5' },
+    { label: 'Tamar 12:00', value: '6' },
+    { label: 'Yoni 12:30', value: '7' },
+    { label: 'Ravid 13:00', value: '8' },
+  ];
+
+  const _today = moment(new Date()).format("YYYY-MM-DD")
+  const _lastDay = moment(new Date()).add(14, 'day').format("YYYY-MM-DD");
+  const [_selectedDate, setSelectedDate] = useState(_today);
+  const [_chosenQueue, setChosenBarber] = useState("Scheduled Queue...")
+  
+
+  const removeBlueStyle = () =>{
+    const day = _today.split('-')[2];
+    const year = _today.split('-')[0];
+    let dates = {}
+    for(let i = 1 ;i < 13 ; i++){
+      const currDate = `${year}-${i}-${day}` === _selectedDate ? "" : `${year}-${i}-${day}`
+      const currDateNext = `2023-${i}-${day}`;
+      if(currDate === "")
+        continue;
+      if(currDate>_lastDay){
+        dates[[currDate]] = {
+          selectedColor: "white",
+          selectedTextColor: "lightgrey"
+        }
+        dates[[currDateNext]] = {
+          selectedColor: "white",
+          selectedTextColor: "lightgrey"
+        }
+      }else{
+        dates[[currDate]] = {
+          selectedColor: "white",
+          selectedTextColor: 'black'
+        }
+      }
+    }
+    return dates
+  }
+  
+
+  const dates = removeBlueStyle();
+  const changeOnDropDown = (item) => {
+    setChosenBarber(item.label);
+    console.log(item.label);
+  }
+
+
+  
+
+
+  const OnBtnPress = () => {
+    navigation.navigate('WorkingDays');
+    
+}
+
+const DISABLED_DAYS = ['Saturday']
+
+const getDaysInMonth =  (month, year, days) => {
+  let pivot = moment().month(month).year(year).startOf('month')
+  const end = moment().month(month).year(year).endOf('month')
+
+  let dates = {}
+  const disabled = { disabled: true }
+  while(pivot.isBefore(end)) {
+    days.forEach((day) => {
+      dates[pivot.day(day).format("YYYY-MM-DD")] = disabled
+    })
+    pivot.add(7, 'days')
+  }
+
+  return dates
+}
+
+const disabled = getDaysInMonth(moment().month(), moment().year(),  DISABLED_DAYS);
+
+  const appointments = {
+    '2022-12-07': [{details: "Men's haircut, 09:30"} 
+    ,{details: "Men's haircut, 10:30"}
+    ,{details: "Men's haircut, 11:00"}
+    ,{details: "Men's haircut, 12:30"}
+    ,{details: "Men's haircut, 13:00"}
+    ,{details: "Men's haircut, 14:00"}
+    ,{details: "Men's haircut, 15:30"}
+    ,{details: "Men's haircut, 16:30"}
+    ],
+    '2022-12-08': [{details: "Men's haircut, 12:30"}],
+    
+  }
+
+  const onPress = (item) => {
+    console.log(item);
+
+  }
+  
+
+  return (
+    <View style={styles.container}>   
+
+
+    <SafeAreaView style={styles.container_Agenda}>
+      <Fragment>
+      <Agenda
+        minDate={_today}
+        maxDate={_lastDay}
+        hideExtraDays={true}
+        initialDate={_today}
+        items={{
+        ...appointments,
+        }}
```

Comment:
no need for {{...appoinments,}} if your'e not gonna add anything, stick to items={appointments}